compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/Constants.java

```diff
@@ -55,13 +54,13 @@ public class Constants {
     public static final String NILABLE_FLOAT = "float?";
     public static final String NILABLE_DECIMAL = "decimal?";
     public static final String NILABLE_BOOLEAN = "boolean?";
-    public static final String NILABLE_MAP_OF_JSON = "map<json>?";
+    public static final String NILABLE_MAP_OF_ANYDATA = "map<anydata>?";
     public static final String NILABLE_STRING_ARRAY = "string[]?";
     public static final String NILABLE_INT_ARRAY = "int[]?";
     public static final String NILABLE_FLOAT_ARRAY = "float[]?";
     public static final String NILABLE_DECIMAL_ARRAY = "decimal[]?";
     public static final String NILABLE_BOOLEAN_ARRAY = "boolean[]?";
-    public static final String NILABLE_MAP_OF_JSON_ARRAY = "map<json>[]?";
+    public static final String NILABLE_MAP_OF_ANYDATA_ARRAY = "map<json>[]?";
```

Comment:
```suggestion
    public static final String NILABLE_MAP_OF_ANYDATA_ARRAY = "map<anydata>[]?";
```
should be `anydata` right?

Reply:
Yes, will fix that 👍🏼